docs/advanced_topics/settings.rst

```diff
@@ -268,6 +268,24 @@ This setting lets you override the maximum number of pixels an image can have. I
 This setting enables feature detection once OpenCV is installed, see all details on the :ref:`image_feature_detection` documentation.
 
 
+Documents
+---------
+
+.. code-block:: python
```

Comment:
Can we add a link from the [`before_serve_document`](https://docs.wagtail.io/en/v2.6.1/reference/hooks.html#document-serving) hook documentation, warning that this won't be called if `WAGTAILDOCS_SERVE_METHOD = 'direct'`?

Reply:
Good shout! Added in d09445eedffab911acc40e2d9567ffa7963b594e.